---
title: Schedule
tags: 
keywords: schedule
summary: "Quick summary of the expected schedule of classes"
sidebar: main_sidebar
permalink: schedule.html
toc: false
folder: main
---

## Schedule
This full course is expected to follow a logical flow to completion. This begins with Introductory classes, then several coding languages, and lastly workflow specific classes. Below is a table listing the individual classes as well as anticipated pre-requisite classes/skills. 

| Class Name | Expected Pre-requisite classes/skills | Material | Expected Completion Knowledge |
|-------|--------|---------|----------|
| [Introducton to Next Generation Sequencing](ngs_landing.html) | Basic molecular biology knowledge | Lectures: Intro to NGS | Understanding what NGS is, how the various technologies work, and why you might use NGS |
| [Introduction to Bioinformatics](bi_landing.html) | Completetion of Intro to NGS | Lectures: Intro to Bioinformatics | Understanding what Bioinformatics is in the context of NGS |
| [Introduction to Cloud and HPC resources](cloud-hpc_landing.html) | Completion of Intro to NGS and Bioinformatics  | Lectures: Intro to Cloud and HPC resources | Understanding why you might use Cloud or HPC resources, what options are available, and how to access them |
| [Introduction to Unix](unix_landing.html) | Completion of Intro to Bioinformatics and basic computer skills | Lectures: Intro to Unix. Notebooks: Unix Basics | Understanding the Unix command line interface and bash, as well as capability to perform basic CLI operations |
| [Introduction to Python](python_landing.html) | Completion of Intro to Bioinformatics and Intro to Unix | Lectures: Intro to Python. Notebooks: Python Basics, Python Data Wrangling | Understanding the Python language, how to access python, and capability to perform basic python operations |
| [Introduction to R](R_landing.html) | Completion of Intro to Bioinformatics and Intro to Unix | Lectures: Intro to R. Notebooks: R Basics | Understanding the R langauge, how to access R, and capability to perform basic R operations |
| [Whole Genome Sequencing 101](wgs_landing.html) | All non-language Introduction classes and Introduction to Unix | Lectures: WGS Library preparations, WGS alignment theory, WGS BWA, WGS De novo assembly. Notebooks: WGS BWA, WGS De novo assembly | Understanding how to create a WGS library, how to bioinformatically align reads to a reference, how BWA works and how to use BWA in an analysis workflow, and how de novo assembly works and how to use de novo assembly in an analysis workflow |
| [RNA Sequencing 101](rna_landing.html) | All Introduction classes and WGS alignment theory  | Lectures: RNA-Seq library preparations, RNA-Seq Mapping and Alignment, RNA-Seq Differential Gene Expression, RNA-seq Pathways Analysis, RNA-seq Clustering and Principal Components. Notebooks: RNA-Seq STAR, RNA-Seq Salmon, RNA-seq DESeq2, RNA-seq GO, RNA-Seq Clustering, RNA-seq Plotting, RNA-seq to completion | Understanding how to make an RNA-seq library, how to bioinformatically align or map the reads to a reference, perform differential gene expression comparisons, perform gene ontology or other pathways analyses, cluster samples, and generate standard RNA-seq plots |
| [Epigenetic Sequencing 101](epi_landing.html) | All Introduction classes, WGS alignment theory, and RNA-seq differential gene expression theory | Lectures: Epigenetic Library Preparations, Epigenetics Mapping and Peak calling, Epigenetics Integration with other -omics sets. Notebooks: Epigenetics MACS2, Epigenetics Integration | Understanding how to generate an epigenetics library prep, how to align reads and perform peak calling, and how to integrate epigentic data with other -omics data sets such as RNA-seq |
| [Single Cell Sequencing 101](singe-cell_landing.html) | All Introduction classes, WGS alignment theory, RNA-seq class | Lectures: Single Cell library preparations, Single Cell Standard Workflows, Single Cell Multi-Omics, Single Cell Pseudotime Analysis. Notebooks: Single Cell Single Sample, Single Cell Multi-sample integration, Single Cell Hash Tagging, Single Cell CITE-Seq, Single Cell Pseudotime | Understanding how single cell library preparations are made, how to optimize samples upstream for single cell workflows, how to perform single cell analysis for single samples, multiple samples, hash tags, CITE-seq, and with pseudotime analysis |
| [Long Read Sequencing 101](long-read_landing.html) | All Introduction classes, WGS class | Lectures: Long Read Library Preparations, Long Read Phylogenetics. Notebooks: PacBio basic processing, Oxford Nanopore basic processing, Phylogenetics | Understanding the options for long read sequencing and how they operate, how to generate long read library preparations, how to perform basic processing of long read data, and how to generate phylogenetic output from long reads |
| [Metagenomics 101](meta_landing.html) | All Introduction classes, WGS class, RNA-seq class, Long Reads class | Lectures: Metagenomics Library Preparations, 16s rRNA based Metagenomics, Shotgun Metagenomics. Notebooks: 16s workflow, shotgun workflow | Understanding how to generate a metagenomics based library prep, why 16s rRNA based or shotgun based is more applicable, and how to perform basic processing of 16s rRNA based workflows or shotgun workflows. |
| [Workflow Management and Containers](workflow-containers_landing.html) | All Introduction classes, recommended at least one workflow specific class | Lectures: - Workflow managers, container tools. Assignments: Generating a full workflow with containerization | Understanding the different options of workflow management, the usage of containerized tools for reproducibility, and how to generate a full workflow for a chosen data set |

